site/src/components/input/station_dropdown_menu/index.tsx

```diff
@@ -29,11 +32,14 @@ type StationDropdownMenuProps = {
 }
 
 export const StationDropdownMenu = (props: StationDropdownMenuProps) => {
-  const favorites = useFavorites(state => ({
-    add: state.addFavorite,
-    remove: state.removeFavorite,
-    has: state.isFavorite
-  }))
+  const favorites = useFavorites(
+    state => ({
+      add: state.addFavorite,
+      remove: state.removeFavorite,
+      has: state.isFavorite
+    }),
+    shallow
+  )
 
   const isFavorite = favorites.has(props.currentStation)
```

Comment:
This change introduces a bug. Since functions don't trigger a re-render `favorites.has()` will use the previous state unless something else triggers a re-render.

Reply:
Fixed in https://github.com/jqpe/junat.live/pull/268/commits/bf11b43acea4fbc59e094ef475dbbdf24bd6417a